Infra: remove `globals.tfvars` and describe `ci-config` (#180)
populationgenomics/hail: infra/gcp/README.md

   use_artifact_registry = false
   ```

> - You can optionally add a `ci_config` section into the `.tfvars` file, 
>   to enable CI triggered by git events:

Reviewer: ```suggestion
- You can optionally add a `ci_config` section to `$HOME/.hail/global.tfvars`, 
  to enable CI triggered by git events:
```

    
   ```
   ci_config = {
     github_oauth_token = "<TOKEN>"
     github_user1_oauth_token = "<TOKEN>"
     # This `false` here will prevent the CI from merging PRs into the repo:
     watched_branches = [["broadinstitute/hail:main", true, false]]  
     deploy_steps = ["deploy_batch", "test_batch_0", "deploy_ci"]
     bucket_location = "<gcp-zone>"
     bucket_storage_class = "MULTI_REGIONAL"
     bucket_name = "hail-ci"
     github_context = "ci-gcp"
   }
   ``` 
  

>   Because of the secret values for `github_oauth_token` and `github_user1_oauth_token`,

Reviewer: ```suggestion
  Because of the sensitive values for `github_oauth_token` and `github_user1_oauth_token`,
```

>   you may want to avoid adding the `.tfvars` file under the version control. To share 

Reviewer: ```suggestion
  you should never put `$HOME/.hail/global.tfvars` unencrypted under the version control. To share 
```
